apps/full-site-editing/full-site-editing-plugin/dotcom-fse/class-full-site-editing.php

```diff
@@ -23,7 +23,7 @@ class Full_Site_Editing {
 	 *
 	 * @var array
 	 */
-	private $template_post_types = [ 'wp_template_part' ];
+	private $template_post_types = array( 'wp_template_part' );
```

Comment:
uhhhhhh phpcs, what you doing here?

Reply:
Back in the day I was trying to make it consistent with shorthand syntax, but if `phpcs` wants `array` by default I'm fine with that, as long as we don't have to fight it on each PR. 😄

Reply:
Definitely. Ideally, we'd let phpcs allow `[  ]`, but probably not worth it for this PR